SMLHelper/Assets/Spawnable.cs

```diff
@@ -128,14 +157,14 @@ public void Patch()
 
             CorePatchEvents.Invoke();
 
-            this.IsPatched = true;
+            IsPatched = true;
 
             OnFinishedPatching?.Invoke();
         }
 
         internal virtual void PatchTechType()
         {
-            this.TechType = this.TechTypeHandler.AddTechType(ModName, this.ClassID, this.FriendlyName, this.Description, false);
+            TechType = TechTypeHandler.AddTechType(ModName, ClassID, FriendlyName, Description, false);
```

Comment:
Be careful with Code Cleanup. Filling the diff with "non-changes" like makes it harder to spot the changes that actually matter.